Linux/Car_Client/chronoscomm.cpp

```diff
@@ -874,7 +891,7 @@ bool ChronosComm::decodeMsg(quint16 type, quint32 len, QByteArray payload, uint8
                 rcmm.command = vb.vbPopFrontUint8();
                 break;
             default:
-                qDebug() << "RCMM: Unknown value id:" << QString::number(value_id, 16);
```

Comment:
Is there a reason for not including this in the terminal printout?

Reply:
Yes and no.:) It **should** be there (I will put it back, thanks for noticing!), I removed it for debugging purposes since Maestro is sending an obsolete value id (status). If no one protest I can make the changes in Maestro and keep this printout.